docker/centos7/centos7.Dockerfile

```
@@ -40,3 +46,6 @@ RUN PLATFORM=$(uname -m | sed 's/_/-/') \
 
 # Inject UUID
 RUN sed -i "s/00000000-0000-0000-0000-000000000000/$(cat /proc/sys/kernel/random/uuid)/" "$PHP_EXT_DIR/relay.so"
+
+# Ensure relay was installed properly.
+RUN php -m | grep relay
```

```suggestion
RUN php --ri relay
```
proper extension info, as in the other Dockerfile

---

`grep relay` will find "barelaying" too 🙃